operator/pkg/model/helpers.go

```diff
@@ -108,3 +110,24 @@ func hostnameMatchesWildcardHostname(hostname, wildcardHostname string) bool {
 	wildcardMatch := strings.TrimSuffix(hostname, strings.TrimPrefix(wildcardHostname, allHosts))
 	return len(wildcardMatch) > 0
 }
+
+func GetInternalListenerCIDRs(ipv4, ipv6 bool) []*envoy_config_core.CidrRange {
```

Comment:
Is there any import from cilium/cilium where this function could be located instead so that it would not need to be replicated in both the agent and the operator?